api/go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/ceph/ceph-csi-operator/api
+
+go 1.22.0
+
+toolchain go1.22.5
+
+require (
+	k8s.io/api v0.30.3
+	k8s.io/apimachinery v0.30.3
+	sigs.k8s.io/controller-runtime v0.18.4
+)
+
+require (
+	github.com/go-logr/logr v1.4.1 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
You should not have 2 required sections inside a mode file. Please merge both into 1

Reply:
this is auto generated, if i run go mod tidy
the require gets split again

Reply:
There we will 2 require one for direct dependencies and another for indirect dependencies

Reply:
Ack, thanks for the clarification